cmd/evmosd/testnet.go

```diff
@@ -324,7 +342,15 @@ func initTestnetFiles(
 			return err
 		}
 
+		const CreateValidatorTxGasLimit = 300000
```

Comment:
where does this value come from?

Reply:
I executed MsgCreateValidator by starting the genesis block and found that the gas it consumed was about 213551, so I set it to 300000. In fact, the gasMeter type of all transactions in the genesis block is InfiniteGasMeter, not BasicGasMeter, so in theory it is okay to set it to 0, but it cannot be infinite. Make sure that the user's assets aevmos in the genesis block are enough to deduct gas fee. Set it to 300000 here I just make it look like a normal consumption value.
<img width="1476" alt="gas-comsume" src="https://github.com/evmos/evmos/assets/9688029/beb6ae3b-f254-47cf-b6ef-d594a17dab24">